fury/primitive.py

```diff
@@ -851,3 +851,235 @@ def prim_cylinder(radius=0.5, height=1, sectors=36, capped=True):
         triangles = (np.array(triangles).reshape(2 * sectors, 3))
 
     return vertices, triangles
+
+
+def prim_para_mobius_strip():
+    """Return vertices and triangle for a Möbius strip
+
+    Returns
+    -------
+    vertices: ndarray
+        vertices coords that compose our Möbius strip
+    triangles: ndarray
+        triangles that compose our Möbius strip
+    """
+
+    npoints = 100
+    sin = np.sin
+    cos = np.cos
+    u = np.linspace(0, 2*np.pi, npoints)
+    v = np.linspace(-1, 1, npoints)
+    u, v = np.meshgrid(u, v)
+    u = u.reshape(-1)
+    v = v.reshape(-1)
+    points2D = np.vstack([u, v]).T
+    tri = Delaunay(points2D)
+    triangles = tri.simplices
+
+    x = (1 + v/2 * cos(u/2)) * cos(u)
+    y = (1 + v/2 * cos(u/2)) * sin(u)
+    z = v/2 * sin(u/2)
+    xyz = np.vstack([x, y, z]).T
+    vertices = np.ascontiguousarray(xyz)
+    return vertices, triangles
+
+
+def prim_para_kleins_bottle():
+    """Return vertices and triangle for Klein bottle
+
+    Returns
+    -------
+    vertices: ndarray
+        vertices coords that compose our Klein bottle
+    triangles: ndarray
+        triangles that compose our Klein bottle
+
+    """
+
+    npoints = 100
```

Comment:
same comment as above: can you put it as a function parameter with `100`as default value

Reply:
done